ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-backplane-1.0.yaml

```diff
@@ -84,6 +90,33 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ocm-ci-image-mirror
+- as: publish-test
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: backplane-operator-test
+    env:
+      IMAGE_REPO: backplane-operator-test
+      REGISTRY_ORG: open-cluster-management
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/open-cluster-management"
+        export OSCI_PIPELINE_PRODUCT_PREFIX=backplane
+        export OSCI_PIPELINE_REPO=backplane-pipeline
+        export SELF="make -f Makefile.prow"
+        export OSCI_PUBLISH_DELAY="0"
```

Comment:
The publish delay should probably be 10 minutes to give it time and spread out the publishes